setup.py

```diff
@@ -32,21 +32,13 @@ def get_long_description():
 
 
 def main():
-    version = sys.version_info[:2]
     virtualenv_open = ['virtualenv>=1.11.2']
-    virtualenv_capped = ['virtualenv>=1.11.2,<14']
     install_requires = ['py>=1.4.17', 'pluggy>=0.3.0,<1.0', 'six']
     extras_require = {}
     if has_environment_marker_support():
-        extras_require[':python_version=="2.6"'] = ['argparse']
-        extras_require[':python_version=="3.2"'] = virtualenv_capped
         extras_require[':python_version!="3.2"'] = virtualenv_open
```

Comment:
This section can be removed and `virtualenv_open` can be added directly to `install_requires` above.